library/Fission/User/Mutate.hs

```diff
@@ -71,6 +74,29 @@ create' username password email herokuUUID = do
       logInfo $ "Inserted user " <> display uID
       return $ Right uID
 
+updatePassword :: MonadRIO    cfg m
+               => MonadSelda      m
+               => HasLogFunc  cfg
+               => ID User
+               -> User.Password
+               -> m (Either Error.Create User.Password)
```

Comment:
💅 Question for ya: how do you feel about this style:

```haskell
updatePassword 
  :: ( MonadRIO    cfg m
     , MonadSelda      m
     , HasLogFunc  cfg
     )
  => ID User
  -> User.Password
  -> m (Either Error.Create User.Password)
```

This is both a more common style, and after a few months and two onboardings, I think possibly clearer than the row of `=>`. Very clear where the constraints are versus arguments.

Dropping the `::` to the next line is newer, but starting to catch on in the community, too. It makes indentation more consistent.

I was thinking about us slowly changing to this style as we go. Any thoughts?

Reply:
Already left my thoughts in the other PR :wink: but yes big fan